src/auxinfo/auxinfo_commit.rs

```diff
@@ -23,7 +23,7 @@ pub(crate) struct AuxInfoCommit {
 impl AuxInfoCommit {
     pub(crate) fn from_message(message: &Message) -> Result<Self> {
         if message.message_type() != MessageType::Auxinfo(AuxinfoMessageType::R1CommitHash) {
-            return bail!("Wrong message type, expected MessageType::Auxinfo(AuxinfoMessageType::R1CommitHash)");
+            return Err(crate::errors::InternalError::MisroutedMessage);
```

Comment:
Instead of writing out the full crate path, we should import crate::errors::InternalError so we can just write `InternalError::MisroutedMessage` instead.